src/utils/navigation.ts

```diff
@@ -208,22 +209,22 @@ export const NAVIGATION: Navigation = {
   [config.BENEFICIAL_OWNER_TYPE_URL]: {
     currentPage: config.BENEFICIAL_OWNER_TYPE_PAGE,
     previousPage: () => config.BENEFICIAL_OWNER_STATEMENTS_URL,
-    nextPage: [config.CHECK_YOUR_ANSWERS_URL, config.TRUST_INFO_URL]
+    nextPage: [config.CHECK_YOUR_ANSWERS_WITH_PARAMS_URL, config.TRUST_INFO_URL]
```

Comment:
i think we only need to change the next page to a 'wiyth params' version if the config is for a 'with_params' page, this one is BENEFICIAL_OWNER_TYPE_URL which is the old type so we need that to go to next page without params, so leave as config.CHECK_YOUR_ANSWERS_URL.
same applies for the other changes, only change the next page if the config is for a 'with_params' page

Reply:
edited according to above.